tests/response_synthesizers/test_google.py

```diff
@@ -20,14 +22,22 @@
 if has_google:
     import llama_index.vector_stores.google.generativeai.genai_extension as genaix
 
-    genaix.set_defaults(
+    genaix.set_config(
```

Comment:
Nit: Maybe use set_google_config and GoogleConfig instead?

Reply:
Done.

Reply:
undid this because of the imports issue :)